### DISCUSSION TITLE
Installing Argo Workflows Cluster wide with Postgres and Minio enabled

### DISCUSSION THREAD

**Discussion by cboin1996** [argoproj/argo-workflows#8110] [Q&A]
Hello, I noticed there is a cluster wide installation manifest on the repository, however this does not include minio or postgres.

Furthermore, it seems that the quick start installation is namespaced. Is it possible for me to install clusterwide with minio and postgres enabled?

(Tagging my colleague on this for visibility. @blairdrummond)

**Comment by terrytangyuan** [comment]
The minio or postgres manifests are not included in either namespace or cluster wide installation. They are meant to be used for quick start examples to showcase different features. You should be able to include those additional minio/postgres manifests as part of your installation.

**Reply by cboin1996** [reply]
Thank you for the quick response! Is there a recommended way of aggregating everything together (Im pretty new to kubernetes, just want to make sure I am going about the install the best way possible).

**Reply by terrytangyuan** [reply]
You can just merge any manifests into one YAML file and separate them by `---`. Or you can run `kubectl apply` on them separately.

**Comment by alexec** [comment]
I don't recommend using Kubernetes for stateful apps like databases and buckets.
MinIO is intended to simulate S3. If you're using AWS, use S3. If you're using GCP or something else, use whatever they provide.
Likewise Postgres should be Aurora.
